<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>623</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuVideoInput">
    <property name="title">
     <string>VideoInput</string>
    </property>
    <addaction name="actionWebcam"/>
    <addaction name="actionFile"/>
   </widget>
   <widget class="QMenu" name="menuDistanceMeasure">
    <property name="title">
     <string>DistanceMeasure</string>
    </property>
    <addaction name="actionEuclidean"/>
    <addaction name="actionSSD"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionReset"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Network"/>
   </widget>
   <widget class="QMenu" name="menuTransformation">
    <property name="title">
     <string>Transformation</string>
    </property>
    <addaction name="actionRotation"/>
    <addaction name="actionScale"/>
   </widget>
   <widget class="QMenu" name="menuSearch_Using">
    <property name="title">
     <string>Search using...</string>
    </property>
    <addaction name="actionAll_ARBs"/>
    <addaction name="actionLatest_Highest_RL"/>
    <addaction name="actionAbove_Avarage_RL"/>
    <addaction name="actionLatest_Connected"/>
    <addaction name="actionHighest_Connected"/>
    <addaction name="separator"/>
    <addaction name="actionPredict_Location"/>
   </widget>
   <widget class="QMenu" name="menuThresholds">
    <property name="title">
     <string>Thresholds</string>
    </property>
    <addaction name="actionOT"/>
    <addaction name="actionST"/>
   </widget>
   <addaction name="menuVideoInput"/>
   <addaction name="menuDistanceMeasure"/>
   <addaction name="menuTransformation"/>
   <addaction name="menuThresholds"/>
   <addaction name="menuSearch_Using"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionWebcam">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Webcam</string>
   </property>
  </action>
  <action name="actionFile">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>File</string>
   </property>
  </action>
  <action name="actionEuclidean">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Euclidean</string>
   </property>
  </action>
  <action name="actionSSD">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>SSD</string>
   </property>
  </action>
  <action name="actionCC">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>CC</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print Number of ARBs</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionUse_Predicted_Area">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use Predicted Area</string>
   </property>
  </action>
  <action name="actionRotation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Rotation</string>
   </property>
  </action>
  <action name="actionScale">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Scale</string>
   </property>
  </action>
  <action name="actionSave_Network">
   <property name="text">
    <string>Save Network</string>
   </property>
  </action>
  <action name="actionUse_Most_Likely_ARBs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use Most Likely ARBs</string>
   </property>
  </action>
  <action name="actionPredict_Location">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Predict Location</string>
   </property>
  </action>
  <action name="actionAll_ARBs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>All ARBs</string>
   </property>
  </action>
  <action name="actionLatest_Highest_RL">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Latest + Highest RL</string>
   </property>
  </action>
  <action name="actionAbove_Avarage_RL">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Above Avarage RL</string>
   </property>
  </action>
  <action name="actionOT">
   <property name="text">
    <string>OT</string>
   </property>
  </action>
  <action name="actionST">
   <property name="text">
    <string>ST</string>
   </property>
  </action>
  <action name="actionLatest_Connected">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Latest + Connected</string>
   </property>
  </action>
  <action name="actionHighest_Connected">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Highest + Connected</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
